helpers/password_test.go

```diff
@@ -0,0 +1,36 @@
+package helpers_test
+
+import (
+	"github.com/markgravity/golang-ic/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Password", func() {
+	Describe("HashPassword", func() {
+		Context("given a string", func() {
+			It("returns the hashed string", func() {
+				Expect(helpers.HashPassword("hello password")).To(ContainSubstring("$"))
+			})
+		})
+	})
+
+	Describe("ComparePassword", func() {
+		Context("given a correct password", func() {
+			It("returns nil", func() {
+				hashedPassword := "$2a$10$9rnyG9shT0T49i9CBQcnYuICoTwgKvwFCY/EEET63PqjJTat1qHRW"
+				correctPassword := "123456"
+				Expect(helpers.ComparePassword(hashedPassword, correctPassword)).To(BeNil())
+			})
+		})
+
+		Context("given an incorrect password", func() {
+			It("returns error", func() {
```

Comment:
```suggestion
			It("is NOT nil", func() {
```
This would align with the way it is asserted `NotTo(BeNil())`. :white_check_mark: 
And we can use "it is nil" for the test above.